src/crimson/osd/pg_backend.cc

```diff
@@ -80,47 +80,53 @@ PGBackend::load_metadata(const hobject_t& oid)
     coll,
     ghobject_t{oid, ghobject_t::NO_GEN, shard})).safe_then_interruptible(
       [oid](auto &&attrs) -> load_metadata_ertr::future<loaded_object_md_t::ref>{
-	loaded_object_md_t::ref ret(new loaded_object_md_t());
-	if (auto oiiter = attrs.find(OI_ATTR); oiiter != attrs.end()) {
-	  bufferlist bl = std::move(oiiter->second);
-	  ret->os = ObjectState(
-	    object_info_t(bl, oid),
-	    true);
-	} else {
-	  logger().error(
-	    "load_metadata: object {} present but missing object info",
-	    oid);
-	  return crimson::ct_error::object_corrupted::make();
+        loaded_object_md_t::ref ret(new loaded_object_md_t());
+        if (auto oiiter = attrs.find(OI_ATTR); oiiter != attrs.end()) {
+          bufferlist bl = std::move(oiiter->second);
+          ret->os = ObjectState(
+            object_info_t(bl, oid),
+            true);
+        } else {
+          logger().error(
+            "load_metadata: object {} present but missing object info",
+            oid);
+          return crimson::ct_error::object_corrupted::make();
```

Comment:
in general, i'd avoid blending formatting changes into functional ones.

Reply:
:+1: A dedicated commit would be perfect.

Reply:
5184d2903beddee04bb16ff6d9995a5b90693fb2